infinispan-remote/persistence/sql-store/src/main/java/org/infinispan/tutorial/simple/remote/persistence/DBCreator.java

```diff
@@ -0,0 +1,146 @@
+package org.infinispan.tutorial.simple.remote.persistence;
+
+import org.apache.commons.io.IOUtils;
+
+import java.nio.charset.StandardCharsets;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DBCreator {
+
+   static final String JDBC_DRIVER = "org.h2.Driver";
+   static final String USER = "infinispan";
+   static final String PASS = "secret";
+   private String DB_URL;
+   private org.h2.tools.Server server;
+
+   public void startDBServer() {
+      startDBServer("9123");
+   }
+
+   public void startDBServer(String port) {
+      DB_URL = String.format("jdbc:h2:tcp://localhost:%s/~/library;", port);
+      try {
+         server = org.h2.tools.Server.createTcpServer("-tcpPort", port, "-tcpAllowOthers", "-ifNotExists").start();
+      } catch (SQLException throwables) {
+         throwables.printStackTrace();
+      }
+   }
+
+   public void stopDBServer() {
+      if (server != null) {
+         server.stop();
+      }
+   }
+
+
+   public void createAndPopulate() {
+      Connection conn = null;
+      Statement stmt = null;
+      try {
+         Class.forName(JDBC_DRIVER);
+         System.out.println("Connecting to database...");
+         conn = DriverManager.getConnection(DB_URL,USER,PASS);
+         System.out.println("Creating table in given database...");
+         stmt = conn.createStatement();
+         String sqlCreate = IOUtils.toString(DBCreator.class.getClassLoader().getResource("initDB.sql"), StandardCharsets.UTF_8);
+         String sqlInsert = IOUtils.toString(DBCreator.class.getClassLoader().getResource("populateDB.sql"), StandardCharsets.UTF_8);
+         System.out.println("Created table in given database...");
+         stmt.executeUpdate(sqlCreate);
+         System.out.println("Populate database...");
+         stmt.executeUpdate(sqlInsert);
+         stmt.close();
+         conn.close();
```

Comment:
These lines aren't needed as they are in the finally below.

Reply:
yes